mesa/visualization/templates/js/HexDraw.js

```diff
@@ -131,24 +132,25 @@ var HexVisualization = function(width, height, gridWidth, gridHeight, context, i
         text_color: Color of the inscribed text.
         */
 	this.drawHex = function(x, y, radius, color, fill, text, text_color) {
-		var cx = (x + 0.5) * cellWidth;
+		const cx = (x + 0.5) * cellWidth;
+		let cy;
 		if(x % 2 == 0){
-			var cy = (y + 0.5) * cellHeight;
+			cy = (y + 0.5) * cellHeight;
 		} else {
-			var cy = ((y + 0.5) * cellHeight) + cellHeight/2;
+			cy = ((y + 0.5) * cellHeight) + cellHeight/2;
 		}
 		maxHexRadius = cellHeight/Math.sqrt(3)
-		var r = radius * maxHexRadius;
+		const r = radius * maxHexRadius;
 
 		function hex_corner(x,y, size, i){
-		    var angle_deg = 60 * i
-		    var angle_rad = Math.PI / 180 * angle_deg
+		    const angle_deg = 60 * i
+		    const angle_rad = Math.PI / 180 * angle_deg
 		    return [(x + size * Math.cos(angle_rad) * 1.2),
 		            (y + size * Math.sin(angle_rad))]
 		}
 
 
-		var px, py
+		let px, py
 		context.beginPath();
 		[px, py] = hex_corner(cx,cy,r,1)
```

Comment:
`const [px, py] = hext_corner(...)`

Reply:
Oh, `[px, py]` are redeclared later. In thase case, `let [px, py] = hex_corner(...)`